balancer-js/src/lib/utils/time.ts

```diff
@@ -0,0 +1,83 @@
+import { sub } from 'date-fns';
+
+export const oneSecondInMs = 1000;
+export const oneMinInMs = 60 * oneSecondInMs;
+export const oneHourInMs = 60 * oneMinInMs;
+export const oneDayInMs = 24 * oneHourInMs;
+export const oneWeekInMs = 7 * oneDayInMs;
+
+export const oneSecond = 1;
+export const oneMinInSecs = 60 * oneSecond;
+export const oneHourInSecs = 60 * oneMinInSecs;
+
+export const twentyFourHoursInMs = 24 * oneHourInMs;
+export const twentyFourHoursInSecs = twentyFourHoursInMs / oneSecondInMs;
+
+export const timeNowInMs = Math.floor(Date.now() / oneSecondInMs);
+
+export const oneYearInSecs = twentyFourHoursInSecs * 365;
+export const oneWeekInSecs = twentyFourHoursInSecs * 7;
+
+export function dateTimeLabelFor(date: Date): string {
+  return date.toLocaleString(undefined, {
+    weekday: 'short',
+    year: 'numeric',
+    month: 'short',
+    day: 'numeric',
+    timeZoneName: 'short',
+    hour: '2-digit',
+    minute: '2-digit',
+    second: '2-digit',
+  });
+}
+
+export function toJsTimestamp(unixTimestamp: number): number {
```

Comment:
good to have it extracted, i think there might be some other places where we could use that. can we limit the lib just to `toJsTimestamp` / `toUnixTimestamp`? is anything else used anywhere?

Reply:
ok